### PR TITLE
Add current and expected json files and code for edx.bookmark.added event

### PR DESCRIPTION
**Pull request by mumarkhan999**
**Trello Link:** [Click here](https://trello.com/c/qBJtEc65/59-bookmark-event-edxbookmarkadded)

**Description:** Contains current and expected json files for edx.bookmark.added event

**Checks before merge:**

 

- [x] JSON Reviewed
- [x]   Code Reviewed
- [x]   Unit Test Pass
- [x]   Commits squashed

### CODE REVIEW

**Comment by asamolion** [comment]
There are unnecessary JSON files in both current and expected directories in this PR. Please clean those directories accordingly.

**Comment by asamolion** [comment]
There should be only your own files.

**Comment by mumarkhan999** [comment]
> There should be only your own files.

Deleted

File: openedx/features/caliper_tracking/tests/expected/edx.bookmark.added.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "id": "https://localhost:18000/u/ali",
+    "name": "ali",
+    "type": "Person"
+  },
+  "action": "Bookmarked",
+  "object": {
+    "id": "http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course/courseware/d8a6192ade314473a78242dfeedfbf5b/edx_introduction/1?activate_block_id=block-v1%3AedX%2BDemoX%2BDemo_Course%2Btype%40vertical%2Bblock%40vertical_0270f6de40fc",
+    "type": "Page"
+  },
+  "event_time": "2018-12-07T14:24:26.143Z",
```
Comment (danialmalik) [inline openedx/features/caliper_tracking/tests/expected/edx.bookmark.added.json]: `eventTime` is the correct key name. Please correct this.

File: openedx/features/caliper_tracking/tests/current/edx.bookmark.added.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (asamolion) [inline openedx/features/caliper_tracking/tests/current/edx.bookmark.added.json]: Please sort the values alphabetically.

File: openedx/features/caliper_tracking/tests/expected/edx.bookmark.added.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  "type": "AnnotationEvent",
+  "actor": {
+    "id": "http://localhost:18000/u/ali",
+    "name": "ali",
+    "type": "Person"
+  },
+  "action": "Bookmarked",
+  "object": {
+    "id": "http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course/courseware/d8a6192ade314473a78242dfeedfbf5b/edx_introduction/1?activate_block_id=block-v1%3AedX%2BDemoX%2BDemo_Course%2Btype%40vertical%2Bblock%40vertical_0270f6de40fc",
+    "type": "Page"
```
Comment (asamolion) [inline openedx/features/caliper_tracking/tests/expected/edx.bookmark.added.json]: Move the key/value pairs `course_id`, ` bookmark_id` and `component_usage_id` to the `extensions` key inside `object`

**Comment by asamolion** [comment]
@arooshaarif Please review.

File: openedx/features/caliper_tracking/tests/current/edx.bookmark.added.json (hunk 2 above)
Comment (mumarkhan999) [reply openedx/features/caliper_tracking/tests/current/edx.bookmark.added.json]: done

File: openedx/features/caliper_tracking/tests/expected/edx.bookmark.added.json (hunk 3 above)
Comment (mumarkhan999) [reply openedx/features/caliper_tracking/tests/expected/edx.bookmark.added.json]: done.
I have also moved the component_type also as it is also related to object.

**Comment by asamolion** [comment]
Squash and merge.

**Pull request merged**